### ISSUE TITLE
Do not delete the backup files when uninstalling OH via a package manager

### ISSUE THREAD

**Issue by nelsonaponte** [openhab/openhab-linuxpkg#229]
When openHAB is managed by a package manager (apt, yum, dnf) and the user attempts to remove (uninstall) openHAB using the package manager, the folder under /var/lib/openhab/backups/ and all of its content should not be deleted.

If possible, a warning message saying "The backups folder under /var/lib/openhab/backups/ was not automatically deleted so you don't lose data if you want to reinstall openHAB. Please, delete it manually if required." should be displayed once the remove process ended successfully.

**Comment by BClark09** [comment]
Agreed that the removal of openhab should not delete backups at least without prior warning. At the very least, the backup script should warn the user that a directory outside the package created directories should be used.

I'm away for a couple of weeks - but I'll look into the best options soon after.
